xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ endif()
 if(ARCH MATCHES arm)
   list(APPEND SOURCES yuv2rgb.neon.S)
   list(APPEND HEADERS yuv2rgb.neon.h)
+  set_source_files_properties(yuv2rgb.neon.S PROPERTIES COMPILE_FLAGS -mfpu=neon)
```

Comment:
`COMPILE_FLAGS` is deprecated. See https://cmake.org/cmake/help/latest/prop_tgt/COMPILE_FLAGS.html

Reply:
And congrats on your first contribution!

Reply:
Thx for the review, I will change for COMPILE_DEFINITIONS